libraries/chain/db_maint.cpp

```diff
@@ -1061,25 +1057,13 @@ void process_hf_868_890( database& db, bool skip_check_call_orders )
          }
       } // end loop of each feed
 
-      // if any feed was modified, print a warning message
-      if( feeds_changed )
-      {
-         wlog( "Found invalid feed for asset ${asset_sym} (${asset_id}) during hardfork core-868-890",
-               ("asset_sym", current_asset.symbol)("asset_id", current_asset.id) );
```

Comment:
I guess we've found something, so this function can't be removed? Please update the TODO above to a comment explaining why this function can't be removed.

Reply:
Yes, something was found. But the TODO below indicates more can be cleaned up. I am going to have to take a step back and build a better understanding of HF 935. Thanks for the careful review. More to come.

Reply:
I have verified that this is not needed, and has been removed. More cleanup in the works.